s3_project/classes/applicant_info_class.py

```diff
@@ -0,0 +1,68 @@
+from s3_project.classes.extraction_class import ExtractFromS3
+import boto3
+import json
+from datetime import datetime
+
+
+class ApplicantInfoClean(ExtractFromS3):
+
+    def __init__(self):
+        super().__init__()
+        self.iterate()
+
+    # This method iterates through each file, and applies the cleaning methods to each file.
+    # This method also appends the cleaned files to a dictionary.
+    def iterate(self):
+        object_dict = {}
+        talent_json_dict = {}
+        for file in self.talent_json_list:
+            s3_object = self.s3_client.get_object(Bucket=self.bucket_name, Key=file)
+            body = s3_object['Body'].read()
+            object_dict = json.loads(body)
```

Comment:
I thought the extraction is defined already in the ExtractFromS3 - why is it re-coded here?

Reply:
The extraction class extracts the filename as a key, in this method we access the body of that key